create_data/README.md

```diff
@@ -0,0 +1,32 @@
+# Generating test data
+
+The test data can be generated by building the docker image defined and running it:
+
+```bash
+docker build -t matrix_data .
+docker run -v /path/to/data:/data matrix_data --data-yaml /data/data.yaml --data-path /data
```

Comment:
This looks a bit confusing, is there a way to make it less complicated?
(I probably should try to run it and then it might make more sense?!)

Reply:
Yeah it's not super straightforward. I'll add a little more explanation.